server/monitor.go

```diff
@@ -2259,11 +2259,12 @@ type JSInfo struct {
 	Disabled bool            `json:"disabled,omitempty"`
 	Config   JetStreamConfig `json:"config,omitempty"`
 	JetStreamStats
-	StreamCnt    int          `json:"total_streams,omitempty"`
```

Comment:
This is ok to change since I believe that these were added just in main and not in public release.

Reply:
Yes exactly..